magma/conversions.py

```diff
@@ -200,30 +203,34 @@ def convertbits(value, n, totype, checkbit):
             Direction.InOut: InOut
         }[T.direction](Bit)
 
-    value = totype[len(Ts), T](ts)
+    value = totype[len(Ts), T](ts, name=name)
     if n is not None and len(value) < n:
+        # TODO: The extended value isn't named, but perhaps we'd like to move
+        # to an explicit convert + extend rather than doing them in a single
+        # operation? If so, then we could provide the same name interface for
+        # the extension operators
```

Comment:
nit
```suggestion
        # TODO(leonardt): The extended value isn't named, but perhaps we'd like to move
        # to an explicit convert + extend rather than doing them in a single
        # operation? If so, then we could provide the same name interface for
        # the extension operators.
```